app/controllers/AuthenticatedAction.scala

```diff
@@ -23,37 +23,41 @@ import models.request.AuthenticatedRequest
 import models.{Operator, Permission, Role}
 import play.api.mvc.Results._
 import play.api.mvc._
-import play.api.{Configuration, Environment, Logger}
+import play.api.{Configuration, Environment, Logging}
 import uk.gov.hmrc.auth.core.AuthProvider.PrivilegedApplication
 import uk.gov.hmrc.auth.core._
-import uk.gov.hmrc.auth.core.retrieve.{Credentials, Name, Retrievals, ~}
+import uk.gov.hmrc.auth.core.retrieve.{~}
+import uk.gov.hmrc.auth.core.retrieve.v2.Retrievals
```

Comment:
This fixes deprecation warnings about `Retrievals`. The `credentials` and `name` are now optional so we must handle the case where we can't retrieve them.
This also required updating the Wiremock responses in the integration tests to fix the property names.